src/checkErrors.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable max-len */
+const errors = [
+  {
+    message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  },
+  {
+    message: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  },
+  {
+    message: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+  },
+];
+
+const checkErrors = (url, toCase) => {
+  const errorMessage = [];
+
+  if (!url.length) {
+    errorMessage.push(errors[0]);
+  }
+
+  if (!toCase) {
+    errorMessage.push(errors[1]);
+  }
+
+  if (!['SNAKE', 'KEBAB', 'PASCAL', 'CAMEL', 'UPPER'].includes(toCase) && toCase) {
```

Comment:
Move it to the variable
```suggestion
  if (!....includes(toCase) && toCase) {
```